test/wpt/status/streams.json

```diff
@@ -29,8 +36,8 @@
     "skip": "Browser-specific test"
   },
   "readable-streams/owning-type.any.js": {
+    "note": "Readable streams with type owning are not yet supported",
```

Comment:
Please put the note inside the "fail" below where it belongs. :-)

Reply:
Oops .. my mistake : ( 
I fixed it right away and pushed the correction ! 
Thank you for the feedback :-)